docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '2'
+services:
+  fpm:
+    hostname: fpm.magento2.docker
+    image: shiftedreality/cloud-docker-php:7.1-fpm
+    ports:
+      - 9000
+    links:
+      - db
+    volumes_from:
+      - appdata
```

Comment:
Docker Compose's `volume` syntax should eliminate the need for an additional third party containers or using `volumes_from` here

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes

Reply:
It's using to avoid copy-paste

Reply:
@JacobBrownAustin you cannot have service without an image. They're using in few places, so that's why separated.